packages/amazonq/src/lsp/client.ts

```diff
@@ -30,14 +31,16 @@ export function startLanguageServer(extensionContext: vscode.ExtensionContext, s
         ],
     }
 
+    const serverPath = executablePaths.lsp
+
     // If the extension is launch in debug mode the debug server options are use
     // Otherwise the run options are used
     let serverOptions: ServerOptions = {
         run: { module: serverPath, transport: TransportKind.ipc },
         debug: { module: serverPath, transport: TransportKind.ipc, options: debugOptions },
     }
 
-    const child = cp.spawn('node', [serverPath, ...debugOptions.execArgv])
+    const child = cp.spawn(executablePaths.node, [serverPath, ...debugOptions.execArgv])
```

Comment:
if we can find a way to use our ChildProcess module, then after https://github.com/aws/aws-toolkit-vscode/pull/6304 we get instrumentation for free :)